src/markd/options.cr

```diff
@@ -56,7 +56,7 @@ module Markd
       @prettyprint = false,
       @emoji = false,
       @tagfilter = false,
-      @base_url = nil
+      @base_url = nil,
```

Comment:
I saw this rule failed, but don't know why this only failed on macOS ...

Reply:
Linux may have failed too but CI was being wonky